### PR TITLE
Allow building dist kernels with savedconfig on loong

### PR DESCRIPTION
**Pull request by xen0n**
As a first experimental step towards proper dist kernel support.

EDIT: I've tested building and booting 6.6.7 on real hardware (Loongson 3A6000 and 2-way 3C5000L) with sd-boot and UKI-fied kernels. Everything works fine and I'll use the config as a starting point for the dist kernel config...

### CODE REVIEW

**Comment by xen0n** [comment]
`dist-kernel-utils.eclass` also needs an addition.

File: eclass/kernel-build.eclass [hunk 1]
```diff
@@ -203,6 +203,12 @@ kernel-build_src_configure() {
 			.config)
```
Comment (mgorny) [inline eclass/kernel-build.eclass:203]: The summary line of the commit message is too long (it wraps).

**Review by mgorny — APPROVED** [review]
LGTM, pending that summary fix. That said, I don't really understand why you're modifying multiple kernel versions but keywording only one of them but that's a minor point.

File: eclass/kernel-build.eclass (hunk 1 above)
Comment (xen0n) [reply eclass/kernel-build.eclass:203]: I've reworded the summary line.

**Pull request closed without merging**